components/atoms/SVGIcon/svg-icon.tsx

```diff
@@ -11,17 +11,26 @@ interface IconProps {
 
 const SVGIcon: React.FC<IconProps> = ({ onClick, className, IconImage, size = 16 }) => {
   return (
-    <div className={clsx(`inline-flex`, className)} onClick={onClick}>
-      <svg>
-        <use
-          href={IconImage}
-          width={size}
-          height={size}
-          className="items-center justify-center"
-          style={{ stroke: "currentcolor", maxWidth: "100%", height: "auto" }}
-        />
-      </svg>
-    </div>
+    <svg
```

Comment:
This should be an actual button that houses the SVG, and we can remove the `tabIndex`, `onKeyDown`, and `role="button" from the SVG and promote the `onClick` to the button.